lib/watchbuild/options.rb

```diff
@@ -39,7 +39,13 @@ def self.available_options
         FastlaneCore::ConfigItem.new(key: :sample_only_once,
                                      description: 'Only check for the build once, instead of waiting for it to process',
                                      is_string: false,
-                                     default_value: false)
+                                     default_value: false),
+        FastlaneCore::ConfigItem.new(key: :slack_url,
+                                     short_option: '-n',
```

Comment:
I wouldn't include a short option here, it wouldn't be very clear when reviewing build scripts, and looks more like it would be short for a generic `--notify`.